dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -174,7 +194,10 @@ object DistriOptimizer {
           driverMetrics.add("computing time for each node", computingTime)
 
           val finishedThreads = trainingThreads.filter(!_.isCancelled).map(_.get())
-          recordsNum += finishedThreads.size * tensorBuffer.head._2.size(1)
+          recordsNum += finishedThreads.size * (batchBuffer.head._2 match {
```

Comment:
The previous name was tensorBuffer. I changed it to batchBuffer because the variable type for MiniBatch has been changed to support Activity. How about dataBuffer?